glue/main.py

```diff
@@ -90,7 +103,7 @@ def load_plugins(splash=None, require_qt_plugins=False):
             _loaded_plugins.add(item.module)
```

Comment:
I can't easily comment on lines above but based on my previous comment,

```
     if item.module in REQUIRED_PLUGINS:
                raise
            elif item.module in REQUIRED_PLUGINS_QT and require_qt_plugins:
                raise
```

should then become:

```
     if item.module in plugins_to_require:
                raise
```